### PR TITLE
Development Merging

### PR DESCRIPTION
**Pull request by IR-Savvy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,7 +191,7 @@
   }
   </style>
 </head>
-<body id="myPage" data-spy="scroll" data-target=".navbar" data-offset="60" data-is="123" data-name="abc" data-name="abc" data-name="abc">
```
Comment (Osama-Moveit) [inline index.html:193]: this one is ok

**Review by Osama-Moveit — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Syed-IR) [reply index.html:193]: Okay

File: index.html [hunk 2]
```diff
@@ -190,7 +191,7 @@
...
+<body id="myPage" data-spy="scroll" data-target=".navbar" data-offset="60" data-conflict="This is going to be a pain in the a..">
```
Comment (Osama-Moveit) [inline index.html:194]: need to revert this

**Review by Osama-Moveit — CHANGES_REQUESTED** [review]

**Review by IR-Savvy — COMMENTED** [review]
Re-check please

**Pull request merged**